pkgs/by-name/lh/lha/package.nix

```diff
@@ -7,27 +7,30 @@
 
 stdenv.mkDerivation {
   pname = "lha";
-  version = "unstable-2021-01-07";
+  version = "unstable-2024-11-27";
 
   src = fetchFromGitHub {
     owner = "jca02266";
     repo = "lha";
-    rev = "03475355bc6311f7f816ea9a88fb34a0029d975b";
-    sha256 = "18w2x0g5yq89yxkxh1fmb05lz4hw7a3b4jmkk95gvh11mwbbr5lm";
+    rev = "26b71be85a762098bdeb95f4533045c7dad86f31";
+    hash = "sha256-jiYTBqDXvxTdrvHYaK+1eo4xIpl+B9ZljhBBYD0BGzQ=";
   };
 
   nativeBuildInputs = [ autoreconfHook ];
 
-  meta = with lib; {
+  meta = {
     description = "LHa is an archiver and compressor using the LZSS and Huffman encoding compression algorithms";
-    platforms = platforms.unix;
-    maintainers = [ maintainers.sander ];
+    platforms = lib.platforms.unix;
+    maintainers = with lib.maintainers; [
+      sander
+      momeemt
+    ];
     # Some of the original LhA code has been rewritten and the current author
     # considers adopting a "true" free and open source license for it.
```

Comment:
nit: the difference in LHa from `meta.description` vs LhA in this comment. Mostly just to be consistent with whatever is more correct.

Reply:
Thank you for pointing out. I have fixed it.